src/store/index.js

```diff
@@ -1,17 +1,12 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
+import { search } from './modules/search'
 
 Vue.use(Vuex)
 
+
 export default new Vuex.Store({
```

Comment:
Use `createStore` instead of  `Vuex.Store`. Because you are creating the global store with modules, not only one.
Please, check the documentation: https://vuex.vuejs.org/guide/modules.html

Reply:
My bad, it's for latest version. Everything is correct